src/commands/deploy/spec/cbd.spec.server.ts

```diff
@@ -200,45 +200,41 @@ describe('sasjs cbd having stream app', () => {
     jest.resetAllMocks()
   })
 
-  it(
-    `should deploy compile and build with streamConfig`,
-    async () => {
-      const appLoc = `/Public/app/cli-tests/${target.name}`
-      const appLocWithSpaces = `${appLoc}/with some/space s`
-      await updateLocalTarget(target.name, {
-        appLoc: appLocWithSpaces,
-        streamConfig,
-        jobConfig: undefined,
-        serviceConfig: undefined,
-        deployConfig: {
-          deployServicePack: false,
-          deployScripts: [`sasjsbuild/${target.name}.sas`]
-        }
-      })
-
-      const command = new Command(`cbd -t ${target.name} -f`.split(' '))
-      await expect(compileBuildDeployServices(command)).toResolve()
-
-      const logFileContent = await readFile(
-        path.join(__dirname, appName, 'sasjsbuild', `${target.name}.log`)
-      )
-
-      const appLocTransformed = '/with%20some/space%20s'
-      const streamingApplink = `${target.serverUrl.replace(
-        'https://',
-        ''
-      )}/SASJobExecution?_FILE=${appLoc}${appLocTransformed}/services/${
-        streamConfig.streamServiceName
-      }.html&_debug=2`
-
-      expect(logFileContent.replace(/\s/g, '')).toEqual(
-        expect.stringContaining(streamingApplink)
-      )
-
-      await updateLocalTarget(target.name, { appLoc })
-    },
-    2 * 60 * 1000
```

Comment:
Arbitrary timeouts like this are the only thing that has been removed. GitHub shows the whole block as changed because the indentation has changed.

Reply:
We already have a longer timeout defined in https://github.com/sasjs/cli/blob/main/jest.server.config.js#L5. These tests are overriding it with a shorter timeout, which is not guaranteed to work all the time.